### DISCUSSION TITLE
Time series tooltip should hide null values

### DISCUSSION THREAD

**Discussion by theadzik** [grafana/grafana#47380] [Dashboards, Panels & Visualizations]
I have a Time series panel with Tooltip enabled. It shows memory utilization of Kubernetes nodes:
![2022-04-06_11-06-07](https://user-images.githubusercontent.com/138645/161939408-b0895a94-d0ea-462f-92fc-bdff89bdd6f9.png)

Since AKS can scale up and down automatically some nodes are really short-lived. The problem is that if a node was present for any period of time it will show on a tooltip at all points.

I would like to have an option for the tooltip to show only non-null values from the highlighted time frame.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
